test/cmdlineTests.sh

```diff
@@ -93,6 +93,27 @@ function compileFull()
     fi
 }
 
+function ask_expectation_update()
+{
+    local newExpectation="${1}"
+    local expectationFile="${2}"
+    while true;
+    do
+        set +e
+        read -t10 -p "(u)pdate expectation/(q)uit? "
```

Comment:
There's an additional whitespace at the end.

Reply:
It is intentional, I copied the string from isoltest https://github.com/ethereum/solidity/blob/ef3a18999c840c523cce73f6074e816334b5c753/test/tools/isoltest.cpp#L179